rental.rb

```diff
@@ -0,0 +1,15 @@
+class Rental
+  def initialize(date, book, person)
+    @date = date
+
+    @book = book
+    book.rentals.push(self)
+
+    @person = person
+    person.rentals.push(self)
+  end
+
+  attr_accessor :date
+
+  attr_reader :book, :person
```

Comment:
- [ ] great job on the associations 👍  for `@book` and `@person` you might want to set up both the setter and the getter as well. you can refer to [this article](https://github.com/microverseinc/curriculum-ruby/blob/main/oop/articles/oop_principles_examples.md#useful-shortcuts-attr_reader-attr_writer-and-attr_accessor) for more details.